README.md

```diff
@@ -4,3 +4,26 @@ Installers for MNE-Python for macOS, Windows, and Linux.
 
 ⚠️ Nothing to see here for now, please move along and check back again in a few
 days. ⚠️
+
+
+
+## Installation
+
+To install MNE-Python simply double click the installer.
+You can customise some parameters, but the default values should suit most users.
+Once completed, a conda evironment should be installed in your home directory.
+
+
+## Usage
+
+### MacOS
+
+Move to your home directory and activate the environment:
+
+```bash
+cd ~
+source mne-python/bin/activate
```

Comment:
Would it not be easier to do it without the `cd`?

`source ~/mne-python/bin/activate`
?

btw:
If users already have `conda` installed, they can also run
```bash
conda activate ~/mne-python
```
This will also produce a more obvious prompt prefix
(but I have to admit that asking users to first install Miniconda kind of goes against the entire idea of the installer)

Reply:
> asking users to first install Miniconda kind of goes against the entire idea of the installer

Agreed.